web/modules/custom/dpl_webmaster/src/Form/InstallOrUpdateModule.php

```diff
@@ -291,4 +291,42 @@ public function submitForm(array &$form, FormStateInterface $form_state): void {
     }
   }
 
+  /**
+   * Copy uploaded module into place and run updates.
+   *
+   * @param string[] $projects
+   *   List of projects to update.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   Form state to set redirect on.
+   */
+  protected function redirectToUpdates(array $projects, FormStateInterface $form_state): void {
```

Comment:
Consider the naming of this function.

When you read the documentation it is not totally clear how it relates to redirects.